modules/installing-ocp-agent-ibm-z-kvm.adoc

```diff
@@ -12,6 +12,7 @@ endif::[]
 = Adding {ibm-z-name} agents with {op-system-base} KVM
 
 Use the following procedure to manually add {ibm-z-name} agents with {op-system-base} KVM.
+This procedure is necessary only for {ibm-z-name} clusters with {op-system-base} KVM.
```

Comment:
```suggestion
Only use this procedure for {ibm-z-name} clusters with {op-system-base} KVM.
```

Reply:
I can rephrase to something like that, sure